src/overview/container.js

```diff
@@ -171,12 +172,42 @@ class OverviewContainer extends Component {
         </ResultsMessage>
     )
 
+    renderBadTermNoResult = isBadTerm => (
+        <div>
```

Comment:
This needs to go into a separate view (with a `props.children` `string` rather than `isBadTerm` `bool` flag) rather than container render method.